database/sql/V048__create_settlement_rewards.sql

```diff
@@ -0,0 +1,29 @@
+-- Populated before settlement occurs on chain
+-- (after the submissions have been ranked)
+CREATE TABLE settlement_scores (
+  settlement_id bytea NOT NULL,
+  winning_score numeric(78,0) NOT NULL,
+  reference_score numeric(78,0) NOT NULL
+);
+
+-- Populated after block finalization via transactionReceipt.
+CREATE TABLE settlement_observations (
+  -- the appended settlement ID read from the call data
+  settlement_id bytea NOT NULL,
+  -- the transaction hash and log index to be able to `JOIN` on the `settlement`
+  -- table, read from the transaction receipt
+  tx_hash bytea NOT NULL,
+  log_index bigint NOT NULL,
```

Comment:
Let's make this blog_number and log_index, instead of tx_hash. That is slightly more consistent to how we uniquely identify settlements in other places. `(block_number, log_index)` should be marked as the primary key too.